csharp/ql/test/library-tests/csharp8/PrintAst.expected

```diff
@@ -491,9 +461,6 @@ NullableRefTypes.cs:
 #  131|   16: [Field] l
 #  131|     -1: [TypeMention] MyClass[,,][,,,][][,]
 #  131|       1: [TypeMention] MyClass
-#  131|         1: [TypeMention] MyClass
-#  131|           1: [TypeMention] MyClass
-#  131|             1: [TypeMention] MyClass
```

Comment:
This corresponds to `MyClass?[,,,][][,]?[,,] l;`.

The type mention hierarchy is now similar to `MyClass[,,,][][,][,,] k;` a couple of lines above, that is, there's only one type mention for the multi dimensional array and one for the element type.